styles.css

```diff
@@ -0,0 +1,1240 @@
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+
+#home:hover {
+    background-color: rgb(231 231 232);
+}
+
+#explore:hover {
+    background-color: rgb(231 231 232);
+}
+
+#notifications:hover {
+    background-color: rgb(231 231 232);
+}
+
+#messages:hover {
+    background-color: rgb(231 231 232);
+}
+
+#bookmarks:hover {
+    background-color: rgb(231 231 232);
+}
+
+#lists:hover {
+    background-color: rgb(231 231 232);
+}
+
+#profile:hover {
+    background-color: rgb(231 231 232);
+}
+
+#more11:hover {
+    background-color: rgb(231 231 232);
+}
+
+
+
+
+
+
+
+.tweak {
+    grid-area: tweak;
+    width: 88%;
+    color: white;
+    margin-left: 4%;
+    margin-right: 4%;
+    margin-bottom: 8mm;
+    border-radius: 30px;
+    background-color: rgb(29 156 240);
+    text-align: center;
+    padding-top: 5%;
+    font-size: 18px;
+    font-weight: bolder;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+
+.tweak:hover {
+    background-color: rgb(22 124 192);
+}
+
+.bottom-left {
+    grid-area: bottom-left;
+    margin-bottom: 5mm;
+    padding-top: 2mm;
+    background-color: white;
+    display: grid;
+    grid-template: "pic  info     dots"1fr/ 20% 60% 20%;
+}
+
+#three-dots {
+    max-width: 30px;
+    padding-top: 1mm;
+}
+
+#bottom-profile {
+    grid-area: pic;
+
+}
+
+#bottom-profile img {
+    max-width: 100%;
+    max-height: 100%;
+    object-fit: cover;
+    border-radius: 60%;
+}
+
+#bottom-info {
+    grid-area: info;
+    margin-left: 2mm;
+    margin-top: 2mm;
+}
+
+#bottom-info h4 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bolder;
+    font-size: 4mm;
+}
+
+#bottom-info p {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    font-size: 3.5mm;
+    color: #a2a2a2;
+
+}
+
+#three-dots {
+    grid-area: dots;
+    margin-top: 2mm;
+}
+
+#bottom-info:hover {
+    box-shadow: inset 5em 1em rgb(254 252 13);
+}
+
+
+#bottom-profile img:hover {
+    box-shadow: 5px 5px 10px rgb(207 208 209);
+
+}
+
+#three-dots svg:hover {
+    fill: rgb(22 124 192);
+}
+
+.main-header {
+    grid-area: main-header;
+    background-color: white;
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+    text-align: left;
+    padding-top: 4mm;
+    padding-left: 3mm;
+    display: grid;
+    grid-template-columns: 95% 5%;
+}
+
+.main-header p {
+    font-weight: bolder;
+    font-size: large;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.main-header img {
+    width: 20px;
+    border-radius: 50px;
+}
+
+.main-header img:hover {
+    background-color: rgb(231 231 232);
+}
+
+.middle-main {
+    grid-area: middle-main;
+    background-color: white;
+    border-top: 2px solid rgb(204 215 221);
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+    padding-bottom: 1mm;
+    min-width: 300px;
+    display: grid;
+    grid-template: "pic  main    main"2fr "pic   icons   tweak"1fr/ 10% 1fr 66px;
```

Comment:
This is also a note on readability, but it's easier to read the `grid-template` if you place each row on a new line.